utils/qemu/patches/0007-qga-invoke-separate-applets-for-guest-shutdown-modes.patch

```diff
@@ -1,52 +1,55 @@
-From 80ec6872aceb18c68b1cf5b6f8acd6ad667cbd4f Mon Sep 17 00:00:00 2001
-From: Yousong Zhou <yszhou4tech@gmail.com>
-Date: Thu, 17 Dec 2020 15:55:55 +0800
-Subject: [PATCH] qga: invoke separate applets for guest-shutdown modes
-
-/sbin/shutdown is not available on OpenWrt by default
-
-Origin: "main/qemu: fix shutdown from guest agent"
-https://gitlab.alpinelinux.org/alpine/aports/commit/76b81b486480fd9c3294cd420bcf2df01c27790d
+Origin: community/qemu: fix qemu-guest-agent patch
+https://gitlab.alpinelinux.org/alpine/aports/-/blob/b720d51ec844d4754dd5b29084350aa1f5c9a74d/community/qemu/guest-agent-shutdown.patch
 ---
- qga/commands-posix.c | 5 +++++
- 1 file changed, 5 insertions(+)
-
+diff --git a/qga/commands-posix.c b/qga/commands-posix.c
+index c2bd0b4316..f6dfa20c1f 100644
```

Comment:
This seems to be what causes the checks to fail (in `make package/qemu/refresh`).

Do you plan to fix this and update the commit message? Otherwise I'll just create a similar pull request myself.

Reply:
Please go ahead. Don't have bandwidth this weekend.